docs/images/spi_device_hierarchy.dot

```diff
@@ -29,6 +32,8 @@ digraph classes {
     MCP3202->MCP32xx;
     MCP3204->MCP32xx;
     MCP3208->MCP32xx;
+    MCP3002->MCP3xx2;
+    MCP3202->MCP3xx2;
```

Comment:
Yeah, 'fraid so - but it's the simplest way to implement the 3xx2 protocol on both the 30xx and 32xx branches. Turns out there's nothing complex in this particular case as both sides of the diamond override completely separate methods so there's no complex MRO to figure out.